examples/ckks/bootstrapping/main.go

```diff
@@ -53,7 +56,8 @@ func main() {
 	rotations := btpParams.RotationsForBootstrapping(params.LogN(), params.LogSlots())
 	rotkeys := kgen.GenRotationKeysForRotations(rotations, true, sk)
 	rlk := kgen.GenRelinearizationKey(sk, 2)
```

Comment:
Should we add a `KeyGen` type and a `GenEvaluationKeyForBootstrapping` function for convenience ?

Reply:
Done